mct_quantizers/common/base_inferable_quantizer.py

```diff
@@ -23,9 +23,13 @@ class QuantizationTarget(Enum):
     Weights = "Weights"
 
 
+class InferableQuantizerID(Enum):
+    INFERABLE = "inferable_quantizer_id"
+
+
```

Comment:
Something about the name is weird. If you have an INFERABLE member the name of this enum should be "QuantizerID". What other types do you plan to have here?

Reply:
I don't, that's a kind of singleton, it only has one ID (I didn't want to just use a simple string, but a unique object). I agree that the name is not great, and changing the enum's name to QuantizerID actually sounds good, so I'll do it although it won't include any other ids. Does that seems ok by you?